README.md

```diff
@@ -68,9 +68,8 @@ Remove a selected book.
 
 #### 🔭 Future Features <a name="future-features"></a>
 
-- **Add a book**
-- **Remove a selected booke**
-
+- **Add Mobile vertion**
+- **Add Dark mode**
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you only have the `🔭 Future Features` section once in your README as right it duplicate twice. I suggest you move this, so it comes just after the `Author` section and then add then add it in the `Table of contenets` list.